RELEASE_NOTES.md

```diff
@@ -27,240 +27,185 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
+  area when enabled by using <kbd>Ctrl+click</kbd> on the button.
 
-- Add undo/redo support in lighttable for tags, color labels, rating,
+- Added undo/redo support in lighttable for tags, color labels, ratings,
   metadata, deleted history stack, pasted history stack and applied
-  style.
+  styles.
 
   IMPORTANT:
-  As <kbd>Ctrl-z</kbd> is commonly used for undo/redo this is what is used
-  on the lighttable. This conflicts with the previous <kbd>Ctrl-z</kbd> which
-  was assigned to preview with focus point. The preview (previously <kbd>z</kbd>)
-  and preview with focus detection (previously <kbd>Ctrl-z</kbd>) are now assigned
-  respectively to <kbd>w</kbd> and <kbd>Ctrl-w</kbd>.
+  The 'preview' (previously <kbd>Z</kbd>) and 'preview with focus detection'
+  (previously <kbd>Ctrl+Z</kbd>) actions are now assigned respectively to
+  <kbd>w</kbd> and <kbd>Ctrl+W</kbd> in order to follow the convention
+  of using <kbd>Ctrl+Z<kbd> for the "undo" function.
 
 - A new timeline view has been introduced in the lighttable.
 
-- A new lighttable mode 'culling' is added. It displays a fixed number
-  of consecutive images, starting from the first selected image and
-  allows to pan & zoom them.  It can be navigated with the mouse wheel
-  and keyboard and the number of displayed images can be set with an
-  entry at the bottom.
+- A new 'culling' mode has been added added to the lightable view. It
+  displays a fixed number of consecutive images starting from the first
+  selected, and allows you to pan & zoom them. The number of displayed images
+  can be set by the user, and they can be navigated with the mouse wheel
+  and keyboard.
 
-- A quite extensive rewrite of the lighttable has been made to improve
-  drastically the overall performance. This also includes the
-  filmstrip which has also received a big rewrite for the same reasons. The
-  lighttable is now usable on 4K and 5K monitors.
+- A quite extensive rewrite of the lighttable (including the film strip) has
+  been made to greatly improve the overall performance.The lighttable is now
+  usable on 4K and 5K monitors.
 
-- Add new raster mask supports. This is a copy of a parametric mask
+- Added support for the new 'raster mask', a copy of a parametric mask
   which is stable during the whole pixel-pipe.
 
-- Modules can now be re-ordered with <kbd>Ctrl+Shift+drag</kbd>.
-
-  IMPORTANT PLEASE READ:
-
-  Note that this is NOT a GUI re-ordering but a pixel-pipe
-  reordering. If you're not familiar with pixel pipe order and the
-  actual quality that could result from changing the default, do not
-  change the order.
-
-  The default order has been set to give the best result in the
-  majority of time and will never be on the wrong side.
-
-- The color zones module now display an histogram based on the ‘select
-  by’ channel and displays the selected range if the color picker is
-  in area mode.
-  It acts now like the tone curve (but horizontal),
```

Comment:
This information is lost in the edit.

Reply:
I think I retained the essence of it in the condensed rewording, is there a specific part you think is important but lost?

Reply:
Relating to a horizontal  tone curve.